src/main/webapp/app/overview/course-dashboard/course-exercise-lateness/course-exercise-lateness.component.scss

```diff
@@ -0,0 +1,6 @@
+.tooltip-label-color {
+    width: 15px;
+    height: 15px;
+    display: inline-block;
+    border-radius: 3px;
```

Comment:
As mentioned in other css files, we should ideally avoid using fixed values like this